_config/security.yml

```diff
@@ -0,0 +1,2 @@
+SilverStripe\Security\MemberLoginForm:
```

Comment:
This is a bidirectional coupling between MemberLoginForm and MemberAuthenticator and that feels messy.

Since MemberAuthenticator is responsible for constructing MemberLoginForm, I think it would be better if this became a constructor arg to MemberLoginForm, and `MemberAuthenticator::get_login_form()` can pass `self::class` as an argument.